2022/31xxx/CVE-2022-31121.json

```diff
@@ -1,18 +1,101 @@
 {
```

Comment:
' https://github.com/hyperledger/fabric/releases/tag/v2.4.5' does not match '^(ftp|http)s?://\\S+$'